communitiesuk/funding-service-design-post-award-submit
# SMD-155: add ADDITIONAL_EMAIL_LOOKUPS env var

### Change description
Read ADDITIONAL_EMAIL_LOOKUPS from env var to make it overridable on environment



### How to test
To test locally you can override `FLASK_ENV` app should start successfully e.g. `FLASK_ENV=test flask run`

Will re-test on deployed environment
